File: Daily Challenges/1502. Can Make Arithmetic Progression From Sequence/arithmetic.ts
function canMakeArithmeticProgression(arr: number[]): boolean {
    arr.sort((a,b) => {return a-b});
    const delta = arr[1]-arr[0];
    let ans = true;
    arr.forEach((value,index)=>{
        if (index<arr.length-1 && (arr[index+1]-value)!=delta){
            ans = false;
            return;
        }
    });
    return ans;
};

File: Daily Challenges/1502. Can Make Arithmetic Progression From Sequence/arithmetic.py
class Solution:
    def canMakeArithmeticProgression(self, arr: List[int]) -> bool:
        arr.sort()
        delta = arr[1]-arr[0]

        for x1,x2 in zip(arr[1:],arr[2:]):
            if x2-x1!=delta:
                return False
        return True